docs/concepts/06_messaging.md

```diff
@@ -3,29 +3,30 @@ id: messaging
 title: KILT Messaging
 ---
 
-Distributed trust on the internet only works if you can exchange credentials and other information securely, and be confident that you are not being fooled or eavesdropped on by bad actors.
+Distributed trust on the Internet only works if Credentials and other information can be exchanged securely, and communicating parties can be confident that they are not being fooled or eavesdropped on by bad actors.
 To help with that, KILT provides a **transport-agnostic messaging layer** that helps with securely exchanging data between the respective owners of two DIDs.
 
-This messaging layer provides **authenticated end-to-end encryption** – the gold standard in secure communication – so that you do not need to worry about the security of the technologies you use for transporting the message over the internet – be it sending the encrypted message via email, or posting them to and fetching them from a centralized or decentralized messaging service.
+This messaging layer provides **authenticated end-to-end encryption** – the gold standard in secure communication – in a way that abstracts about the security of the technologies used for transporting the message over the Internet – be it sending the encrypted message via email, or posting them to and fetching them from a centralized or decentralized messaging service.
 
 :::info
 The messaging layer enables secure communication between two digital identities – DIDs.
-A necessary condition for secure communication with a given person or organisation is that you are sure that the DID you are exchanging messages with is really controlled by them.
+A necessary condition for secure communication with a given person or organization is to make sure that the DID on the other side of the communication channel is really controlled by other party, in order to avoid attacks such as *Man in the Middle* (MitM) attacks.
```

Comment:
```suggestion
A necessary condition for secure communication with a given person or organization is to make sure that the DID on the other side of the communication channel is really controlled by the other party, in order to avoid attacks such as *Man in the Middle* (MitM) attacks.
```